src/main/java/org/devcourse/springbasic/validator/VoucherMenuValidator.java

```diff
@@ -0,0 +1,18 @@
+package org.devcourse.springbasic.validator;
+
+import org.devcourse.springbasic.util.RegexPattern;
+import java.util.regex.Matcher;
+
+public class VoucherMenuValidator<T> implements Validator<T> {
+
+    @Override
+    public boolean validate(T valueToVerify) {
+
+        if (valueToVerify instanceof String) {
+            Matcher matcher = RegexPattern.NOT_INPUT_MENU_PATTERN
+                    .matcher(valueToVerify.toString());
+            return matcher.find();
+        }
+        return false;
+    }
```

Comment:
```suggestion
public class VoucherMenuValidator implements Validator<String> {

    @Override
    public boolean validate(String valueToVerify) {
        Matcher matcher = RegexPattern.NOT_INPUT_MENU_PATTERN
            .matcher(valueToVerify);
        return matcher.find();
    }
}

```
이렇게 사용하는게 보다 올바를 것 같습니다~!